### ISSUE TITLE
Add title option

### ISSUE THREAD

**Issue by Gary-Li** [alalonde/angular-scrollable-table#25]
Hi alalonde,
If scrollable-table has an option to allow show/hide the title attribute for each header would be great!
because sometimes we add something that don't want to display the content of header by title way.
E.g. an icon in header cells or using third part tooltip way to display header content.
Thanks!

**Comment by alalonde** [comment]
I agree the options for titles are limited and rather inflexible.  I will look into it if there is sufficient demand.
